### DISCUSSION TITLE
Hollow Clock Powered by Seeed Studio XIAO RA4M1 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1957] [Q&A]
# Hollow Clock Powered by Seeed Studio XIAO RA4M1 | Seeed Studio Wiki

Build a simplp Clock with Seeed Studio XIAO RA4M1.

https://wiki.seeedstudio.com/xiao_ra4m1_clock/

**Comment by sh1ura** [comment]
Link to the original websites and/or movie of the original designer (me) at
https://www.instructables.com/Hollow-Clock-4/
https://www.thingiverse.com/thing:5636482
https://youtu.be/hRpLiRoMx34

You just violate the licensing condition (NC BY-NC-SA) because you do not give credit to the original (BY condition).
In other words, you just steal my design and source code.

**Reply by meilily** [reply]
Dear sh1ura,

This is Lily from Seeed Studio's Community Team. Thank you for bringing this licensing issue to our attention. We sincerely apologize for incorrectly attributing your project. Our initial citation only referenced The Wrench's YouTube video, which was an oversight on our part. We have now properly credited you as the original designer at the beginning of the wiki with links to your original designs.

We have a deep respect for open source licensing and take these obligations seriously. This mistake has helped us improve our attribution practices. Please let us know if any additional changes are needed to ensure full compliance with your project's license.

Best regards,
Team Seeed Studio

**Reply by sh1ura** [reply]
Thank you for quick and perfect response!
